.travis.yml

```diff
@@ -1,3 +1,6 @@
+branches:
+  only:
+  - master
```

Comment:
What happens with pull request builds? pull requests are technically branches (of course not ```master```). Do we get still pull request builds?

I know about an issue with getting the same build twice